test/unit/services/subscription/CdsSubscriberSpec.scala

```diff
@@ -62,6 +63,10 @@ class CdsSubscriberSpec extends UnitSpec with MockitoSugar with ScalaFutures wit
   implicit private val hc: HeaderCarrier                = mock[HeaderCarrier]
   implicit private val mockRequest: Request[AnyContent] = mock[Request[AnyContent]]
 
+  private val injector                  = GuiceApplicationBuilder().injector()
+  implicit val messagesApi: MessagesApi = injector.instanceOf[MessagesApi]
+  implicit val messages: Messages       = MessagesImpl(defaultLang, messagesApi)
```

Comment:
Is this the sort of change you are planning on making to controller specs Patryk?

Reply:
Yes and no :)

I mean it's good to use just Injector (we can copy the trait that we have for Injector) and this is the change that I want to do in the tests.

The second thing that I'm starting is Application as argument in every controller, which makes that unit tests requires application too.

Reply:
I thought of that.  Should I copy the Injector trait over and use that here?  I didn't want to mess with what you might be doing.

Reply:
Yes, you can do it :)

I'm trying to work on the Auth traits, because those traits require this Application